### ISSUE TITLE
Strange message parsing Path with referenced parameters

### ISSUE THREAD

**Issue by cvgaviao** [swagger-api/swagger-parser#1066]
I'm getting a strange message after parsing an openAPi3 file  that smells like a bug:
`attribute paths.'/ads/{retailId}'(get).parameters.There are duplicate parameter values`

Below is a snippet of my spec:
```
paths:  
  /ads/{retailId}:
    parameters:
      - $ref: "#/components/parameters/ParamP_RetailId"
    get:
      parameters:
        - $ref: "#/components/parameters/ParamQ_StartDate"
        - $ref: "#/components/parameters/ParamQ_EndDate"
    
parameters:
    ParamP_RetailId:
      in: path
      name: retailId
      required: true
      schema:
        type: integer
        format: int32
      
    ParamQ_StartDate:
      in: query
      name: startDate
      required: false
      schema:
        type: string
        format: date  
        
    ParamQ_EndDate:
      in: query
      name: endDate
      required: false
      schema:
        type: string
        format: date  
  
```
Am I missing something or is this a bug ?

**Comment by cvgaviao** [comment]
Seems that no matter its location or its name, if there are more than one parameter being referenced inside a operations parameter then the parameter duplication message will be throw.

Also if I remove the path parameter `ParamP_RetailId` then no error are being thrown.

**Comment by cvgaviao** [comment]
I did a debug and found where the error is: `io.swagger.v3.parser.util.OpenAPIDeserializer.getParameter(ObjectNode, String, ParseResult)`.
This method is returning a Parameter object with name and location equals to null. So this explains the duplication message.

**Comment by cvgaviao** [comment]
The pull request [1065](https://github.com/swagger-api/swagger-parser/pull/1065) fixed this issue.
@gracekarina could you please take a look on it ?

**Comment by gracekarina** [comment]
This issue was fixed by #1065
